_docs/Threat-Model.md

```diff
@@ -1,70 +1,126 @@
 ---
 title: Threat Model
 ---
-## Scope of the Threat Model
+#Threat Model
+This model shall provide an overview on the system architecture regarding security. It shall list the attackers and threats Qabel opposes and their capabilities as well as security assumptions. It also shall outline the targeted security goals by describing how they are reached.
 
-The threat model exhaustively lists what an attacker can learn about a user.
-As the computers running a Qabel client or server could be compromised by an
-attacker, the threat model also lists the capabilities of Qabel clients and
-servers.
+##Security Goals
+The main targeted goal of Qabel is to require no trust in any entity in order to achieve **confidentiality**, **integrity**, **authenticity** and partly **anonymity**. The first three goals are reached by using Authenticated end-to-end Encryption. The fourth goal cannot fully be reached due to for example storage write access restriction to the authenticated user who pays for the storage. Thus quota is tracked per user.
```

Comment:
I do like this formulation of the goals and the "Freedom of Trust" notion. Technically the limitation of the currently available anonymization technology also is a problem, e.g. we can't *properly* hide IP addresses.

(Anonymization is a hard problem and there is no real analysis, that the drop box with fake traffic approach works. At least for the time being I would *not* claim that Qabel hides for example information about the sender/recipient of a message.)